src/components/TodoList/TodoList.jsx

```diff
@@ -1,44 +1,79 @@
 import React from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+
 import './TodoList.scss';
 
-export const TodoList = () => (
+export const TodoList = (
+  {
+    statusFilter,
+    handleSelect,
+    handleChange,
+    todos,
+    setUser,
+  },
+) => (
   <div className="TodoList">
+    <select
+      value={statusFilter}
+      onChange={event => handleSelect(event)}
+    >
+      <option value="All">All</option>
+      <option value="Completed">Completed</option>
+      <option value="Not completed">Not completed</option>
+    </select>
+    <input
+      type="text"
+      onChange={event => handleChange(event.target.value)}
+    />
+
     <h2>Todos:</h2>
 
     <div className="TodoList__list-container">
       <ul className="TodoList__list">
-        <li className="TodoList__item TodoList__item--unchecked">
-          <label>
-            <input type="checkbox" readOnly />
-            <p>delectus aut autem</p>
-          </label>
+        {todos.map(todo => (
+          <li
+            key={todo.id}
+            className={classNames({
+              TodoList__item: true,
+              'TodoList__item--unchecked': !todo.completed,
+              'TodoList__item--checked': todo.completed,
+            })}
+          >
+            <label>
+              <input type="checkbox" readOnly />
```

Comment:
if it's checkbox where is its checked attribute?

Reply:
fixed. Now it works as supposed